docs/terminology.md

```diff
@@ -10,17 +12,42 @@ the user can manage a group more easily. Having this given, a group can be
 treated like a single unit. All actions one can apply to a unit using
 `fleetctl`, can also be applied towards a group using `inagoctl`.
 
+There are two types of groups: slice groups and instance groups. Unit files,
+which contain an `@` sign are considered sliceable. If the unit files does **not**
+contain an `@` sign the unit is recognized an instance unit.
+
+Examples:
+
+### Sliceable group
+
+```nohighlight
+└── mygroup
+    ├── mygroup-bar@.service
+    └── mygroup-foo@.service
+```
+
+### Instance group
+
+```nohighlight
+└── timer
```

Comment:
so remove the whole "instance group" part? I must say I din't understand the difference here, @Nesurion ?